modules/doc/content/ncrc/applications/ncrc_hpc_relap5.md

```diff
@@ -1 +1,20 @@
-!template load file=ncrc/applications/ncrc_hpc.md.template ApplicationName=RELAP5 ApplicationLower=relap5 binary=relap5
+# Relap5-3D Binary Access with INL-HPC
+
+Familiarize yourself with [inl/hpc_ondemand.md], and return here with an interactive shell for the
+machine you wish to run your application on. NCRC Application binaries are only available on
+Sawtooth and Lemhi.
+
+## Load Relap5-3D Environment
+
+Logged into either Sawtooth or Lemhi, load the Relap5-3D environment:
```

Comment:
Do we want to give specific machine names? I'm just figuring that this documentation will become out-of-date at some point and we will forget to update it

Reply:
I would love to create a template/extension for {{HPC_MACHINES}} for example, but this request to fix Relap5 was more of an ASAP request.

When the clusters do change, I'll be sure to create this unique feature. There are a lot of places that would need a fix to make this happen.